fe/fe-core/src/main/java/com/starrocks/server/GlobalStateMgr.java

```diff
@@ -1308,6 +1309,13 @@ public long loadAlterJob(DataInputStream dis, long checksum) throws IOException
         return newChecksum;
     }
 
+    // TODO put this at the end of the image before 3.0 release
```

Comment:
You can use RBAC to replace the new adj. The new is not very clear.

Reply:
done